Docs/source/Software/BSP.rst

```diff
@@ -107,12 +107,12 @@ PLL BSP: Sijin Woo
 ==================================
 
 Purpose
-    The PLL is used to increase the clock speed of the STM32F413's clock. Right now, it is set to change it
-    from 16 MHz to 80 MHz.
+    The :term:`PLL <Phase Locked Loop>` is used to increase the clock speed of the STM32F413's clock. 
+    Right now, it is set to change it from 16 MHz to 80 MHz.
 
 Usage
     Calling the init function automatically sets the clock speed to 80 Mhz. In order to verify
-    that it worked, you can call BSP_PLL_GetSystemClock().
+    that it worked, you can call ``BSP_PLL_GetSystemClock()``.
 
 Additional Considerations
     Increasing the clock speed does take more power. Although this is minor considering how much the
```

Comment:
Another additional consideration we should add is that the PLL should be initialized before anything else that is dependent on clock speed (for example, any communications protocols like UART)